src/Symfony/Bundle/TwigBundle/Tests/DependencyInjection/TwigExtensionTest.php

```diff
@@ -208,9 +208,13 @@ public function testStopwatchExtensionAvailability($debug, $stopwatchEnabled, $e
         }
         $container->registerExtension(new TwigExtension());
         $container->loadFromExtension('twig', array());
+
+        $container->register('test_debug_stopwatch', 'stdClass')
+            ->setProperty('stopwatch', new Reference('twig.extension.debug.stopwatch'));
+
         $this->compileContainer($container);
 
-        $tokenParsers = $container->get('twig.extension.debug.stopwatch')->getTokenParsers();
+        $tokenParsers = $container->get('test_debug_stopwatch')->stopwatch->getTokenParsers();
```

Comment:
a better solution would be to use `$container->get('twig')->getExtension(StopwatchExtension::class)` to access the extension, which avoids registering an extra service